Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -21,8 +21,11 @@
 urllib3.disable_warnings()
 
 '''GLOBAL VARS'''
-BLACKLISTED_URL_ERROR_MESSAGE = 'The submitted domain is on our blacklist. ' \
-                                'For your own safety we did not perform this scan...'
+BLACKLISTED_URL_ERROR_MESSAGES = [
+    'The submitted domain is on our blacklist. '
+    'For your own safety we did not perform this scan...',
```

Comment:
```suggestion
    'The submitted domain is on our blacklist. For your own safety we did not perform this scan...',
```

I think this syntax should pass validation and is clearer